src/main/resources/camunda/create_claim.bpmn

```diff
@@ -115,142 +73,301 @@
       <bpmn:incoming>Flow_03t3ygq</bpmn:incoming>
       <bpmn:outgoing>Flow_1mak1f7</bpmn:outgoing>
     </bpmn:serviceTask>
-    <bpmn:serviceTask id="CaseAssignmentToApplicantSolicitor1" name="Assign case to claimant solicitor 1" camunda:type="external" camunda:topic="processCaseEvent">
+    <bpmn:callActivity id="Activity_19rrcw0" name="End Business Process" calledElement="EndBusinessProcess">
+      <bpmn:extensionElements>
+        <camunda:in variables="all" />
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_1mak1f7</bpmn:incoming>
+      <bpmn:incoming>Flow_0iqgyg8</bpmn:incoming>
+      <bpmn:incoming>Flow_1jt9mf9</bpmn:incoming>
+      <bpmn:incoming>Flow_0mdk6wm</bpmn:incoming>
+      <bpmn:outgoing>Flow_08x3izf</bpmn:outgoing>
+    </bpmn:callActivity>
+    <bpmn:serviceTask id="CreateClaimPaymentFailedNotifyApplicantSolicitor1" name="Notify applicant solicitor 1" camunda:type="external" camunda:topic="processCaseEvent">
       <bpmn:extensionElements>
         <camunda:inputOutput>
-          <camunda:inputParameter name="caseEvent">ASSIGN_CASE_TO_APPLICANT_SOLICITOR1</camunda:inputParameter>
+          <camunda:inputParameter name="caseEvent">NOTIFY_APPLICANT_SOLICITOR1_FOR_FAILED_PAYMENT</camunda:inputParameter>
         </camunda:inputOutput>
       </bpmn:extensionElements>
-      <bpmn:incoming>Flow_0oy1b4w</bpmn:incoming>
-      <bpmn:outgoing>Flow_1yheet4</bpmn:outgoing>
+      <bpmn:incoming>Flow_05b1yo7</bpmn:incoming>
+      <bpmn:outgoing>Flow_1av9hkz</bpmn:outgoing>
     </bpmn:serviceTask>
+    <bpmn:serviceTask id="CreateClaimProceedsOfflineNotifyApplicantSolicitor1ForUnRepresentedSolicitor" name="Notify applicant solicitor 1" camunda:type="external" camunda:topic="processCaseEvent">
+      <bpmn:extensionElements>
+        <camunda:inputOutput>
+          <camunda:inputParameter name="caseEvent">NOTIFY_APPLICANT_SOLICITOR1_FOR_RESPONDENT_LITIGANT_IN_PERSON</camunda:inputParameter>
+        </camunda:inputOutput>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_154avkv</bpmn:incoming>
+      <bpmn:outgoing>Flow_03t3ygq</bpmn:outgoing>
+    </bpmn:serviceTask>
+    <bpmn:serviceTask id="IssueClaim" name="Issue Claim" camunda:type="external" camunda:topic="processCaseEvent">
+      <bpmn:extensionElements>
+        <camunda:inputOutput>
+          <camunda:inputParameter name="caseEvent">CLAIM_ISSUE</camunda:inputParameter>
+        </camunda:inputOutput>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_1mjmvjy</bpmn:incoming>
+      <bpmn:outgoing>Flow_0mdk6wm</bpmn:outgoing>
+    </bpmn:serviceTask>
+    <bpmn:serviceTask id="ProceedOflineForUnregisteredFirm" name="Proceed offline (unregistered firm)" camunda:type="external" camunda:topic="processCaseEvent">
```

Comment:
Maybe just capitalise `unregistered` and `firm` for consistency.

Reply:
Done